src/System/Tasks/ConversionRunner.ts

```diff
@@ -6,6 +6,10 @@ import cloneDeep = require("lodash.clonedeep");
 import MarkdownIt = require("markdown-it");
 import checkbox = require("markdown-it-checkbox");
 import emoji = require("markdown-it-emoji");
+import markdownContainer = require("markdown-it-container");
+import markdownInclude = require("markdown-it-include");
+import path = require("path");
```

Comment:
Please remove this and use `upath`s `dirname` instead. It's already imported on line 15:
https://github.com/manuth/MarkdownConverter/blob/f3dde71d9455746939e230887f9ef05b60c9c1f0/src/System/Tasks/ConversionRunner.ts#L15